doc/qbk/3.0.parsing.qbk

```diff
@@ -52,59 +62,98 @@ rest of the elements such as "scheme" below please refer to __rfc3986__:
                     / path-empty
 ```
 
-Each of these functions accepts a __string_view__ and returns a __url_view__
-wrapped in a __result__ type. The following example parses a string literal
-containing a URI:
+The following is an example URI and its main parts:
+
+[teletype]
 ```
-result< url_view > r = parse_uri( "https://www.example.com/path/to/file.txt" );
-
-if( r.has_value() )                         // parsing was successful
-{
-    url_view u = r.value();                 // extract the url_view
-
-    std::cout << u;                         // format the URL to cout
-}
-else
-{
-    std::cout << r.error().message();       // parsing failure; print error
-}
+     foo://example.com:8042/over/there?name=ferret#nose
+     \_/   \______________/\_________/ \_________/ \__/
+      |           |            |            |        |
+scheme     authority       path        query   fragment
 ```
 
+
+For the complete specification please refer to __rfc3986__:
+
+[note
+    This documentation refers to the Augmented Backus-Naur Form (ABNF)
+    notation of
+    [@https://tools.ietf.org/html/rfc2234 rfc2234]
+    to specify particular grammars used by algorithms and containers. While
+    a complete understanding of the notation is not a requirement for using the
+    library, it may help for understanding how valid components of URLs are
+    defined. In particular, this will be of interest to users who wish to
+    compose parsing algorithms using the combinators provided by the library.
+]
+
+[h5 Functions]
```

Comment:
you might just say `[heading Functions]` and let Quickbook auto-number it.

Reply:
No problem. I used `[heading Functions]` first. But changed to `h5` because that's what Boost.Json used.